homeassistant/components/template/binary_sensor.py

```diff
@@ -88,6 +87,8 @@
         vol.Optional(CONF_DEVICE_CLASS): DEVICE_CLASSES_SCHEMA,
         vol.Required(CONF_STATE): cv.template,
         vol.Optional(CONF_UNIT_OF_MEASUREMENT): cv.string,
+        vol.Optional(CONF_DEVICE_ID): selector.DeviceSelector(),
```

Comment:
It seems unrelated to this PR.

Reply:
Yes, I am honestly puzzled here. _I_ did not add this, however when rebasing on dev due to merge conflicts, these were a couple of them. I remembered that something with device assignments to template entities has been added in the last release and assumed that this was the change. Need to double-check that I guess.